blockchain/blockchain.go

```diff
@@ -592,6 +592,17 @@ func (b *Blockchain) WriteBlocks(blocks []*types.Block) error {
 
 		// Update the average gas price
 		b.UpdateGasPriceAvg(new(big.Int).SetUint64(header.GasUsed))
+
+		logArgs := []interface{}{
+			"number", header.Number,
+			"hash", header.Hash,
+			"txns", len(block.Transactions),
+		}
+		if prevHeader, ok := b.GetHeaderByNumber(header.Number - 1); ok {
+			diff := header.Timestamp - prevHeader.Timestamp
+			logArgs = append(logArgs, "generation_time", fmt.Sprintf("%d [s]", diff))
```

Comment:
I think it would be easier to parse and more readable without the ` [s]` part, we will know that the generation_time is always in seconds.

Reply:
I like that it states that it's in seconds. 🙂 
I often find myself wondering whether something is in seconds or milliseconds. If you feel that it is way easier to parse if we omit the unit, I guess it's worth the tradeoff.

Reply:
As discussed, I changed the key to `generation_time_in_sec` and removed unit from the value